ldes-server-domain/src/test/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/shacl/entities/ShaclShapeTest.java

```diff
@@ -0,0 +1,55 @@
+package be.vlaanderen.informatievlaanderen.ldes.server.domain.shacl.entities;
+
+import org.apache.jena.rdf.model.Model;
+import org.apache.jena.rdf.model.ModelFactory;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtensionContext;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.ArgumentsProvider;
+import org.junit.jupiter.params.provider.ArgumentsSource;
+
+import java.util.stream.Stream;
+
+import static org.apache.jena.rdf.model.ResourceFactory.*;
+import static org.apache.jena.rdf.model.ResourceFactory.createResource;
+import static org.junit.jupiter.api.Assertions.*;
+
+class ShaclShapeTest {
```

Comment:
Since your tests do not fail while the model is not included in the equals and hashcode, your tests do not cover all cases :smile:. 

Can you also add tests for the hashcode?

You can check ViewNameTest for a complete example.